src/components/LinkButton.tsx

```diff
@@ -1,31 +1,37 @@
+import { NavLink } from "react-router-dom";
+
 export default function LinkButton({
```

Comment:
Can you add some documentation for what this should be used for? Is it just an abstraction of a button that leads to a certain link?